### PR TITLE
iDefense: Added new integration

### PR DESCRIPTION
**Pull request by yaakovi**
Fixes: demisto/server#9269

### CODE REVIEW

File: Integrations/integration-iDefense.yml [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+  name: url
+  defaultvalue: https://api.intelgraph.verisign.com/
+  type: 0
+  required: true
+- display: API Token
+  name: api_token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
```
Comment (Spivaka) [inline Integrations/integration-iDefense.yml]: proxy?

File: Integrations/integration-iDefense.yml [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    // handle '/' at the end of the url
+    var base_url = params.url.slice(0, params.url.length - params.url.match('/*$')[0].length) + '/rest/';
+
+    function mergeObjects(objs) {
```
Comment (Spivaka) [inline Integrations/integration-iDefense.yml]: move to common

File: Integrations/integration-iDefense.yml [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+        case 'ip':
+            return get_ip(args.ip);
+        case 'url':
+            return get_url(args.url);
+        case 'uuid':
+            // still having issues with this command.
+            return get_uuid(args.uuid);
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            return 'ok';
```
Comment (Spivaka) [inline Integrations/integration-iDefense.yml]: unless I missed something, doesn't seem that you really check something here before returning ok

File: Integrations/integration-iDefense.yml (hunk 1 above)
Comment (yaakovi) [reply Integrations/integration-iDefense.yml]: what do you mean?

Comment (Spivaka) [reply Integrations/integration-iDefense.yml]: we have a flag to use system demisto proxy ettings or general one. you can see it at almost all of other integrations

**Comment by Spivaka** [comment]
you have conflicts, resolve them and merge

**Pull request merged**